### PR TITLE
Abbs 0.7 - Incorporate Gartner and BDM Execs feedback

### PR DESCRIPTION
**Pull request by gcharest**
Gartner:
- Recommended having an exec summary at the very beginning

BDM:
- Needed more clarification on how maintain OSS (should be part of OSS Strategy but can directly point to OSS Framework)
- Needed better clarification on how we acquire the skills to move forward with a solution: mainly in the case where we don't have all the required employees or the employees do not have the current knowledge of a product, what to do then?

### CODE REVIEW

**Review by remyb2canada — COMMENTED** [review]
Gartner's comment suggests putting the Appendix A (diagnostic, or business case) as part of the main document. Interesting :) We put it as an Appendix because executives hate reading lol!

Also, on the Conclusion (line 356, I can't add a comment since it's not part of the PR diff), I would rephrase as such:

With the exception of well-established software for generic or peripheral needs of ESDC, whether buying or building one will translate to work effort. Even traditional services delivered through SaaS, such as email, require considerable configuration efforts for their implementation and continuous maintenance throughout their lifecycle.

The question becomes whether ESDC could and should take on the work effort itself, or seek to procure services to pass those efforts to someone else.

This strategy is intended to…

File: _pages/en/strategy-build-vs-buy.md [hunk 1]
```diff
@@ -40,6 +34,35 @@ permalink: /strategy-build-vs-buy.html
...
 - [Appendix E - Notes and Changelog](#appendix-e---notes-and-changelog)
 
+## Executive Summary
+
+Moving to the digital age requires improving IT's responsiveness and promoting a shared accountability with ESDC stakeholders on the use of technology.
+
+Yet, the current principle when acquiring IT Solutions is to "Buy what we can, Build what we must."
+The assumed reasoning behind this overarching principle is that by purchasing an IT Solution from a third party, risks, responsibilities, and liabilities of its operations are handled by a company who has more resources and technical expertise to produce and maintain software.
+
+This principle is leading to an oversimplified approach that doesn't take into account that, as a government department, ESDC is subject to multiple and fast-paced changes, whether at the [legal, political, organizational or technological levels](https://www.canada.ca/en/employment-social-development/corporate/reports/departmental-plan/2020-2021.html#h2.2-3.2), that impacts its ability to deliver on its mandate.
```
[the remark this answers is not in the document]
Comment (gcharest) [reply _pages/en/strategy-build-vs-buy.md:44]: OK, also based on their comments, I will try to move the "locking ourselves in" up in the paragraph and go on with the rest of the paragraph as to why: Statement followed by argument

File: _pages/en/strategy-build-vs-buy.md [hunk 2]
```diff
@@ -40,6 +34,35 @@ permalink: /strategy-build-vs-buy.html
...
+
+As such, it is critical for the organization to ensure that core-to-mandate or complex IT Solutions remain highly flexible and adaptable to adjust to the pressing changes while also minimizing the risks of those changes by adopting loosely coupled architectures and smaller sized work packages (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html) for proposed ways to support such an approach).
+
+This strategy proposes moving ESDC to the point where all stakeholders understand that IT Solutions are complex and investment decisions have to be taken with flexibility, interchangeability, and interoperability in mind **throughout the entire lifecycle** of an IT Solution, not only at the initial stage.
+
+This strategy complements existing IT initiatives (such as the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx)) as well as supports them with new activities (see [Coherent set of actions](#coherent-set-of-actions)).
```
[the remark this answers is not in the document]
Comment (gcharest) [reply _pages/en/strategy-build-vs-buy.md:58]: Good point, will remove

File: _pages/en/strategy-build-vs-buy.md [hunk 3]
```diff
@@ -40,6 +34,35 @@ permalink: /strategy-build-vs-buy.html
...
+
+This strategy's goal is to clarify the application of the GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and on [Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604) when acquiring new or enhancing existing IT Solutions and IT products within ESDC IITB context.
```
[the remark this answers is not in the document]
Comment (gcharest) [reply _pages/en/strategy-build-vs-buy.md:60]: Ah yes, good point!

**Comment by gcharest** [comment]
@remyb2canada thanks for the feedback!

**Comment by gabrielcossette** [comment]
Great work! Friendly reminder to update the Changelog at the end :)

**Review by gabrielcossette — APPROVED** [review]
Approving as well, so you'll be ready to merge after the pending changes 👍

**Comment by gabrielcossette** [comment]
Oh sorry, didn't notice it was a draft PR. Well, it's going to be pre-approved anyway then, it's on the right track 😉

**Comment by gcharest** [comment]
Closes:
- #1249 
- #1381 
- #1353 
- #1382

**Review by smellems — APPROVED** [review]

**Comment by smellems** [comment]
related #1337
